### DISCUSSION TITLE
pag能否适配安哥拉技术，目前没有找到相关的文档

### DISCUSSION THREAD

**Discussion by lixn12** [Tencent/libpag#2054] [Q&A] [answered]
pag能否适配安哥拉技术，目前没有找到相关的文档，希望有大佬解答

**Comment by kevingpqi123** [comment]
请问安哥拉技术是啥，能否发些技术介绍

**Comment by lixn12** [comment]
pag能否适配angular框架库，目前没有找到相关的文档&nbsp;



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: kevingpqi123 ***@***.***&gt;
发送时间: 2024年1月8日 10:24
收件人: Tencent/libpag ***@***.***&gt;
抄送: lixn12 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [Tencent/libpag] pag能否适配安哥拉技术，目前没有找到相关的文档 (Discussion #2054)





 
请问安哥拉技术是啥，能否发些技术介绍

**Comment by kevingpqi123** [comment, marked as answer]
我们暂时没有计划去做这些， pag 的源码是开源的，如果你们确实需要，可以基于源码去做开发，欢迎参与共建
